autoprof/AP_config.py

```diff
@@ -49,5 +52,5 @@ def set_logging_output(stdout = True, filename = None, **kwargs):
         out_handler.setLevel(kwargs.get("filename_level", logging.INFO))
         out_handler.setFormatter(kwargs.get("filename_formatter", logging.Formatter("%(asctime)s:%(levelname)s: %(message)s")))
         ap_logger.addHandler(out_handler)
-        ap_logger.debug("logging now going to stdout")
+        ap_logger.debug("logging now going to %s" % filename)
```

Comment:
Hey @ConnorStone628 
I changed this to Filename 
since it was in the filename if-block.
If there's any changes to be made, Let me know.